src/main/java/com/buddy/api/domains/account/entities/AccountEntity.java

```diff
@@ -0,0 +1,59 @@
+package com.buddy.api.domains.account.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Table(name = "person")
+public class AccountEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @EqualsAndHashCode.Include
+    @Column(name = "account_id", nullable = false, unique = true)
+    private UUID personId;
+
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
+
+    @Column(name = "email", nullable = false, unique = true)
+    private String email;
+
+    @Column(name = "phone_number")
+    private String phoneNumber;
+
+    @Column(name = "password", nullable = false)
+    private String password;
+
+    @Column(name = "terms_of_user_consent", nullable = false)
+    private Boolean termsOfUserConsent;
+
+    @Column(name = "creation_date", updatable = false)
+    @CreationTimestamp
+    private LocalDateTime creationDate;
+
+    @Column(name = "updated_date")
```

Comment:
```suggestion
    @Column(name = "updated_date", nullable = "false")
```
não tenho certeza se é o caso de que no `INSERT` o `@UpdateTimestamp` vai adicionar a data da inserção, mas se for, então seria interessante colocar a constraint de `NOT NULL` aqui também.

Reply:
O comportamento esperado do @UpdateTimestamp é que ele só gere um valor de timestamp quando houver uma operação de atualização no banco de dados.

Reply:
Se nao tiver deixando, então tem algo errado.

Reply:
pensando melhor, faz sentido sim colocar, desconsidera os comentários haha